tests/unittest/_torch/auto_deploy/unit/singlegpu/transformations/library/test_moe_fusion.py

```diff
@@ -91,19 +170,34 @@ def get_input(self, device):
         return torch.randint(0, 100, (2, 10), device=device)
 
 
-def test_moe_matching():
+@pytest.mark.parametrize(
+    "use_fp8,expected_op,skip",
+    [
+        pytest.param(False, torch.ops.auto_deploy.torch_moe, False, id="simple"),
+        pytest.param(True, torch.ops.auto_deploy.torch_fp8_moe, not fp8_compatible(), id="fp8"),
+    ],
+)
+def test_moe_matching(use_fp8, expected_op, skip):
+    if skip:
+        pytest.skip("FP8 not supported on this hardware")
```

Comment:
Please correctly use `@pytest.mark.skipif(not fp8_compatible(), reason="Requires fp8 support")`